.codeclimate.yml

```diff
@@ -0,0 +1,46 @@
+---
```

Comment:
I see that there are a few changes from the template file in the generator.  Do you plan to add the generic ones ("**.svg", "**.wsdl", etc.) to the template?